frontend/src/pages/GettingStarted.tsx

```diff
@@ -26,90 +27,70 @@ import { commonCss, padding } from '../Css';
-import { Trans } from 'react-i18next';
 
 const DEMO_PIPELINES: string[] = SAMPLE_CONFIG.slice(0, 4);
-
 const DEMO_PIPELINES_ID_MAP = {
   control: 4,
   data: 3,
   tfxKeras: 2,
   tfx: 1,
   xgboost: 0,
 };
-  const PAGE_CONTENT_MD = ({
-    control,
-    data,
-    tfxKeras,
-    tfx,
-    xgboost,
-  }: {
-    control: string;
-    data: string;
-    tfxKeras: string;
-    tfx: string;
-    xgboost: string;}) => 
-  `
-  <br/>
-  ## Build your own pipeline with
-
-    * TensorFlow Extended (TFX) [SDK](https://www.tensorflow.org/tfx/guide) with end-to-end ML Pipeline Template ([Open TF 2.1 Notebook](https://console.cloud.google.com/mlengine/notebooks/deploy-notebook?q=download_url%3Dhttps%253A%252F%252Fraw.githubusercontent.com%252Ftensorflow%252Ftfx%252Fmaster%252Fdocs%252Ftutorials%252Ftfx%252Ftemplate.ipynb))
-    * Kubeflow Pipelines [SDK](https://www.kubeflow.org/docs/pipelines/sdk/)
-
-  <br/>
-
-  ## Demonstrations and Tutorials
-  This section contains demo and tutorial pipelines.
-
-  **Demos** - Try an end-to-end demonstration pipeline.
-
-    * [TFX pipeline demo with Keras](${tfxKeras}) - Classification pipeline based on Keras. [source code](https://github.com/kubeflow/pipelines/tree/master/samples/core/iris)
-    * [TFX pipeline demo with Estimator](${tfx}) - Classification pipeline with model analysis, based on a public BigQuery dataset of taxicab trips. [source code](https://github.com/kubeflow/pipelines/tree/master/samples/core/parameterized_tfx_oss)
-    * [XGBoost Pipeline demo](${xgboost}) - An example of end-to-end distributed training for an XGBoost model. [source code](https://github.com/kubeflow/pipelines/tree/master/samples/core/xgboost_training_cm)
-
-  <br/>
-
-  **Tutorials** - Learn pipeline concepts by following a tutorial.
-
-    * [Data passing in python components](${data}) - Shows how to pass data between python components. [source code](https://github.com/kubeflow/pipelines/tree/master/samples/tutorials/Data%20passing%20in%20python%20components)
-    * [DSL - Control structures](${control}) - Shows how to use conditional execution and exit handlers. [source code](https://github.com/kubeflow/pipelines/tree/master/samples/tutorials/DSL%20-%20Control%20structures)
-
-  Want to learn more? [Learn from sample and tutorial pipelines.](https://www.kubeflow.org/docs/pipelines/tutorials/)
-  `;
 
-  cssRaw(`
-  .kfp-start-page li {
-    font-size: 14px;
-    margin-block-start: 0.83em;
-    margin-block-end: 0.83em;
-    margin-left: 2em;
-  }
-  .kfp-start-page p {
-    font-size: 14px;
-    margin-block-start: 0.83em;
-    margin-block-end: 0.83em;
-  }
-  .kfp-start-page h2 {
-    font-size: 18px;
-    margin-block-start: 1em;
-    margin-block-end: 1em;
-  }
-  .kfp-start-page h3 {
-    font-size: 16px;
-    margin-block-start: 1em;
-    margin-block-end: 1em;
-  }
-  `);
+const PAGE_CONTENT_MD = ({
+  control,
+  data,
+  tfxKeras,
+  tfx,
+  xgboost,
+}: {
+  control: string;
```

Comment:
Are these values still in use?

Reply:
I believe yes, in the  public render(): JSX.Element {...